lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -374,6 +357,30 @@ export class NativeInteractionClient extends BaseInteractionClient {
         return result;
     }
 
+    protected addNativeBrokerFieldsToTelemetry(response: NativeResponse): MATS | null {
```

Comment:
Same as above - Can we please rename this as: this.addTelemetryFromNativeResponse()?

Reply:
Fixed